src/Sulu/Bundle/TrashBundle/Resources/js/containers/RestoreFormOverlay/RestoreFormOverlay.js

```diff
@@ -58,8 +59,15 @@ class RestoreFormOverlay extends React.Component<Props> {
             return;
         }
 
-        const formStore = memoryFormStoreFactory.createFromFormKey(formKey);
-        formStore.loading = true;
+        const formStore = new SchemaFormStoreDecorator(
+            (schema, jsonSchema) => {
+                const store = new MemoryFormStore({}, schema, jsonSchema);
+                store.loading = true;
```

Comment:
setting `store.loading` to `true` in the initializer function instead of the next statement prevents a race condition (loading is finished before next statement is executed) that leads to an infinite loader